templates/vue/src/components/NodeModal.vue

```diff
@@ -284,6 +284,11 @@ export default {
           }
         }
 
+        if (this.node.conditions.length > 0) {
```

Comment:
Conditions existing does not mean that a node will be locked. The conditions would first have to be evaluated to determine if the node will be locked or not. Since this evaluation is done in the backend, you might have to send the request to the backend to do the evaluation and then receive the result of that to update the accessible and unlocked properties here.